### DISCUSSION TITLE
Reverse sorting shortcut - Ctrl/Cmd + click or middle click a list view header

### DISCUSSION THREAD

**Discussion by Serendibb** [burkybang/SuiteAdvanced-Feedback#194] [General]
I've found this from the initial list of function of SuiteAdvanced Feature Documentation. However, I can't get it to work for me. Anybody else have luck with this shortcut please?

**Comment by burkybang** [comment]
This was working up until the NetSuite 2024.2 update. I've had to recreate lots of features for this update. I wasn't aware of this one, so thanks for pointing it out.

**Comment by burkybang** [comment]
This will be fixed in the next release.

**Comment by burkybang** [comment]
This fix is out now! v30.2.0
You can wait for your browser to automatically poll for updates, or you can poll for updates yourself: [How To Update Chrome Extensions](https://www.youtube.com/watch?v=H1NBPr1mmt4)
